modules/porous_flow/test/tests/sinks/s11.i

```diff
@@ -0,0 +1,131 @@
+# Test PorousFlowEnthalpySink boudnary condition
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+  nx = 2
+  ny = 2
+  nz = 2
+  xmin = 0
+  xmax = 10
+  ymin = 0
+  ymax = 10
+  zmin = 0
+  zmax = 10
+[]
+
+[GlobalParams]
+  PorousFlowDictator = dictator
+[]
+
+[UserObjects]
+  [dictator]
+    type = PorousFlowDictator
+    porous_flow_vars = 'pp temp'
+    number_fluid_phases = 1
+    number_fluid_components = 1
+  []
+  [pc]
+    type = PorousFlowCapillaryPressureConst
+    pc = 0.1
+  []
+[]
+
+[Variables]
+  [pp]
+    initial_condition = 1
+  []
+  [temp]
+    initial_condition = 2
+  []
+[]
+
+[Kernels]
+  [mass0]
+    type = PorousFlowMassTimeDerivative
+    fluid_component = 0
+    variable = pp
+  []
+
+  [heat_conduction]
+    type = TimeDerivative
+    variable = temp
+  []
+[]
+
+[Modules]
+  [FluidProperties]
+    [simple_fluid]
+      type = SimpleFluidProperties
+      bulk_modulus = 1
+      density0 = 10
+      thermal_expansion = 0
+      viscosity = 11
+    []
+  []
+[]
+
+[Materials]
+  [ppss]
+    type = PorousFlow1PhaseFullySaturated
+    porepressure = pp
+  []
+  [massfrac]
+    type = PorousFlowMassFraction
+  []
+  [simple_fluid]
+    type = PorousFlowSingleComponentFluid
+    fp = simple_fluid
+    phase = 0
+  []
+  [porosity]
+    type = PorousFlowPorosityConst
+    porosity = 0.125
+  []
+
+  [temperature]
+    type = PorousFlowTemperature
+    temperature = temp
+  []
+[]
+
+[BCs]
+  [left_p]
+    type = PorousFlowSink
+    variable = pp
+    boundary = left
+    flux_function = -1
+  []
+  [left_T]
+    type = PorousFlowEnthalpySink
```

Comment:
Should be a comment here saying: Note, there is no fluid_phase or pressure prescribed, since this is done in the tests file.
Otherwise readers will get confused

Reply:
✅